factory/alchemy.py

```diff
@@ -77,6 +77,10 @@ def _get_or_create(cls, model_class, session, args, kwargs):
                 obj = cls._save(model_class, session, args, {**key_fields, **kwargs})
             except IntegrityError as e:
                 session.rollback()
+
+                if cls._original_params is None:
```

Comment:
This is good!

Please add `_original_params = None` to line  50 in this file.

Reply:
To bad I needed two commits for this

Reply:
Don't sweat it ;) -- @francoisfreitag  is going to make you squash all your commits into 1 commit, so it doesn't matter  😝

Reply:
Looking forward to erase that piece of history!